app/assets/scss/application.scss

```diff
@@ -39,7 +40,6 @@ $govuk-compatibility-govukelements: true;
 @import "node_modules/digitalmarketplace-govuk-frontend/govuk-frontend/all";
 
 // App specific styles
-@import "_footer.scss";
 @import "_text.scss";
 @import "_secondary-action-link.scss";
```

Comment:
Can we remove this? We aren't using secondary-action-link in this repo

Reply:
I think so? This should be in a separate PR though